plugins/plugins/frontend/public/javascripts/countly.views.js

```diff
@@ -979,19 +977,19 @@ window.ConfigurationsView = countlyView.extend({
                 }
                 else {
                     var input = this.getInputByType((id + "." + i).substring(1), "");
-                    var label = this.getInputLabel((id + "." + i).substring(1), i);
+                    label = this.getInputLabel((id + "." + i).substring(1), i);
```

Comment:
In this cases I would prefer variable renaming, mostly because it is hard to follow how was it defined and what is there